### ISSUE TITLE
Currently online is not aware of SCRIPT_HEARTBEAT_FREQUENCY

### ISSUE THREAD

**Issue by haaavk** [milesmcc/shynet#124]
If you increase SCRIPT_HEARTBEAT_FREQUENCY it can break currently online stat.
Right now there is hardcoded 10 seconds. 
```python
currently_online = Session.objects.filter(
      service=self, last_seen__gt=timezone.now() - timezone.timedelta(seconds=10)
).count()
```
I think it could be aware of heartbeat frequency. 
For example 2 * SCRIPT_HEARTBEAT_FREQUENCY

**Comment by CasperVerswijvelt** [comment]
Something like this was fixed in #91, maybe the [is_currently_active](https://github.com/milesmcc/shynet/blob/8be690c417a2fb8774f147a300f26fa5b8d6203c/shynet/analytics/models.py#L60) property can be reused since it is already fixed there?

**Comment by haaavk** [comment]
I'm not sure how to use efficiently model property in queryset. 
Moreover is_currently_active is Session property and Session depends on Service.
It may generate cyclic dependence. 

Maybe there should be a constant for that in core/models. 
For example 
```python
ACTIVE_USER_TIMEDELTA = timezone.timedelta(
     milliseconds=settings.SCRIPT_HEARTBEAT_FREQUENCY * 2
 )
```

**Comment by milesmcc** [comment]
Fixed in v0.9.0
